### ISSUE TITLE
Godot 3 portals that are parallel but not coplanar create a gap in room hulls

### ISSUE THREAD

**Issue by jamie-pate** [godotengine/godot#87685]
### Tested versions

Reproducible in godot 3.5.3, godot 3.5.2. Godot 4.2.1 does not have the portal system and is unaffected.

### System information

Linux - Godot 3.5.3 - GLES3 - Mesa Intel(R) Graphics (ADL GT2)

### Issue description

If you are editing complicated room geometry and you end up with 2 planes that are parallel but not coplanar they may cause a gap between the convex hull of their common room.

![image](https://github.com/godotengine/godot/assets/682658/b3450057-51da-4645-89d4-444eb4e260fc)

Ideally this would create an overlap between `Hall` and `Room3` rather than creating a gap between `Hall2` and `Room3`.

If that's not possible then a warning would be nice.

#### Workaround

To work around this issue ensure all parallel portals that connect to the same face of a room are also coplanar, or rotate the portals to allow creation of a convex hull.

### Steps to reproduce

Create a scene like the  MRP (or download the MRP)
Open the main scene and click 'Convert all rooms' for portal culling. Notice that Room3 and Hall2 have a gap, which causes everything to disappear when the camera is in the gap between rooms after exiting the portal in Hall2

### Minimal reproduction project (MRP)

[portal_room_offset_godot3.zip](https://github.com/godotengine/godot/files/14077488/portal_room_offset_godot3.zip)

**Comment by lawnjelly** [comment]
I remember this scenario being quite difficult at the time, and deciding how to handle it. I agree it could be made more user friendly with regard to small errors in portal placement. Granted in your example you have exaggerated the gap, but there is still the possibility of a flash when going between such small gaps.

I'm wondering also about an alternative - instead of extending `Room3` to meet the portal from `Hall2`, maybe we can just use some sticky logic to count the camera as within `Room3` (even if not strictly speaking within), as this is the most logical choice having gone through that portal.

It's a few years since I worked on this, so I can't guarantee a quick fix (have to make sure not to break another scenario), but I'll take a look. :+1:

**Comment by lawnjelly** [comment]
Still investigating, it turns out that during the original PR I added the ability to select per portal whether it was added to the room bound, then during later changes, this got removed.

The variable `_settings_include_in_bound` exists but is not used. I'll see whether re-adding this helps with problems like these.

It is a great MRP though, I clearly hadn't thought of this situation so will work through a proper fix.
